core/src/main/java/com/novoda/noplayer/PlayerSubtitlesView.java

```diff
@@ -0,0 +1,14 @@
+package com.novoda.noplayer;
+
+import com.google.android.exoplayer2.text.Cue;
+
+import java.util.List;
+
+public interface PlayerSubtitlesView {
```

Comment:
What I don't like about this interface that it is supposed to be used on both implementation of Player but it is coupled with exoplayer. Maybe we could find a way to pass something else than `com.google.android.exoplayer2.text.Cue`